src/dvr/dvr_config.c

```diff
@@ -818,14 +818,14 @@ const idclass_t dvr_config_class = {
     {
       .type     = PT_U32,
       .id       = "removal-days",
-      .name     = N_("DVR File Removal Time (days)"),
+      .name     = N_("DVR File removal period (days)"),
```

Comment:
"DVR *file*...."?

And why are logs retention ("keep for...") while files are removal ("remove after...")? Isn't it the same thing?